app/src/main/java/com/example/timeman/TimerService.kt

```diff
@@ -8,23 +8,34 @@ import java.util.*
 class TimerService: Service() {
     override fun onBind(intent: Intent?): IBinder? = null
 
+    companion object {
+        const val TIMER_UPDATED="timerUpdated"
+        const val TIME_EXTRA="timeExtra"
+
```

Comment:
moved companion object to better fit syntax for code - https://kotlinlang.org/docs/basic-syntax.html